src/screens/TripDetails/components/Time.tsx

```diff
@@ -5,24 +5,43 @@ import {formatToClock} from '@atb/utils/date';
 import React from 'react';
 import {View} from 'react-native';
 import {getTimeRepresentationType, TimeValues} from '../utils';
+import ThemeIcon from '@atb/components/theme-icon';
+import {useTheme} from '@atb/theme';
+import {Realtime as RealtimeDark} from '@atb/assets/svg/color/icons/status/dark';
+import {Realtime as RealtimeLight} from '@atb/assets/svg/color/icons/status/light';
 
-const Time: React.FC<TimeValues> = (timeValues) => {
+const Time: React.FC<{timeValues: TimeValues; showRealtime?: boolean}> = ({
+  timeValues,
+  showRealtime = true,
+}) => {
   const {t, language} = useTranslation();
+  const {themeName} = useTheme();
   const {aimedTime, expectedTime} = timeValues;
   const representationType = getTimeRepresentationType(timeValues);
   const scheduled = formatToClock(aimedTime, language);
   const expected = expectedTime ? formatToClock(expectedTime, language) : '';
 
+  const realtimeIcon = (
+    <ThemeIcon
+      svg={themeName == 'dark' ? RealtimeDark : RealtimeLight}
+      size="small"
+      style={{marginRight: 4}}
+    ></ThemeIcon>
+  );
+
   switch (representationType) {
     case 'significant-difference': {
       return (
         <View style={{flexDirection: 'column', alignItems: 'flex-end'}}>
-          <AccessibleText
-            prefix={t(dictionary.travel.time.expectedPrefix)}
-            testID="expTime"
-          >
-            {expected}
-          </AccessibleText>
+          <View style={{flexDirection: 'row', alignItems: 'center'}}>
+            {showRealtime && realtimeIcon}
+            <AccessibleText
+              prefix={t(dictionary.travel.time.expectedPrefix)}
```

Comment:
As far as I can see `expectedPrefix` says "Forventet tid" and not "Sanntid". Should this be changed?